hazelcast/src/main/java/com/hazelcast/internal/cluster/impl/ClusterClockImpl.java

```diff
@@ -31,14 +37,14 @@
 
     private volatile long clusterTimeDiff;
     private volatile long clusterStartTime = Long.MIN_VALUE;
-    @Probe(level = MANDATORY)
+    @Probe(name = CLUSTER_METRIC_CLUSTER_CLOCK_MAX_CLUSTER_TIME_DIFF, level = MANDATORY)
```

Comment:
Do we need `MS` unit here? The same question for other time probes in this class.

Reply:
Done